packages/metadata-common/src/MetadataEditor.tsx

```diff
@@ -120,7 +123,7 @@ const CodeBlock: React.FC<ICodeBlockProps> = ({
   }, []);
 
   React.useEffect(() => {
-    if (editorRef !== null) {
+    if (editorRef.current !== undefined) {
```

Comment:
On the block above we are checking against null while here we are checking against undefined. What's the differences?

Reply:
`null` and `undefined` are two separate types. I think the original intention was that `undefined` means that a variable hasn't been defined, like `let x;` `x` is `undefined`. Where `null` is a value that represents the absence of an actual value, normally more intensional, like `let x = null;` `x` is `null` here. It's also important to know that `null !== undefined`, but `null == undefined`. `undefined` can also be set, so you can do `let x = undefined;`, for this reason some people try to only use `undefined`, because `undefined` occurs naturally, but `null` doesn't, so if you only use `undefined`  there's normally less corner cases you need to think about. Another thing is that `null` is serializablefor JSON, but `undefined` isn't, so a js object like this: `{ a: "value", b: null }` would serialize to `{ "a": "value", "b": null }`, where `{ a: "value", b: undefined }` would serialize to `{ "a": "value" }`

That was a lot haha...

Now, the reason this is checking for `undefined`, is that `current` might be `undefined`, but can't be `null`. I think the original `null` check on `editorRef` wasn't doing anything